Update DW expression watches concept (#132)
mulesoft/docs-studio: modules/ROOT/pages/dw-expression-watches-view-reference.adoc

> = Evaluate DataWeave Expression View Reference

Reviewer: ```suggestion
= Evaluate DataWeave Expressions
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> The Studio debugger can watch for selected DataWeave expressions in all active breakpoints. If the expression occurs at at any of the active breakpoints, the debugger evaluates the expression, then displays the result in the *Mule Debugger* view.

Reviewer: ```suggestion
The Studio debugger can watch for selected DataWeave expressions in all active breakpoints. If the expression occurs at any of the active breakpoints, the debugger evaluates the expression, then displays the result in the *Mule Debugger* view.
```
Why "can watch"? Does the user have to do something to enable this? Or does the debugger just "watch" on a regular basis?
Who selects the expressions?
Where does evaluation come into this?
Do we mean...:
```suggestion
Select DataWeave expressions and the Studio debugger watches for them in all active breakpoints. If a selected expression appears, the debugger evaluates its and displays the result in the *Mule Debugger* view:
```

image::expression-view-reference.png[]

<1> *Expression field* +

> Write your DataWeave expression in the expression field

Reviewer: ```suggestion
Specify the DataWeave expression to watch for in the expression field.
```

<1> *Clear* +

> Clears all results from the expression evaluations

Reviewer: ```suggestion
Clears all results from the expression evaluations.
```
When would a user want to use this?

<1> *Evaluate* +

> Evaluates the expression from the expression field against the metadata of your flow

Reviewer: ```suggestion
Evaluates the expression you specified in the expression field against the metadata of your flow.
```

<1> *Add to watch* +

> Adds the current DataWeave expression as an expression watch

Reviewer: ```suggestion
Adds the current DataWeave expression as an expression to watch for.
```
In the expression field?
